### PR TITLE
Add Croatian locale to Discourse

### PR DESCRIPTION
**Pull request by markodurdevic**
Please add Croatian locale to Discourse - the translation is at about 40% but hoping to get to at least 60% in the next few days. :)

### CODE REVIEW

File: lib/translations_manager/locales.rb [hunk 1]
```diff
@@ -23,6 +23,8 @@ module TranslationsManager
     'fr',
     'gl',
     'he',
+    'hr',
+    'hr_HR',
```
Comment (gschlager) [inline lib/translations_manager/locales.rb]: This list should only contain `hr`, not `hr_HR`.

**Comment by gschlager** [comment]
Thanks for working on the translations! Are you going to create a PR to add the translations to Discourse core as well?

Please ping me when the translation progress for `core/server.yml` reaches roughly 50% and I'll be happy to help you add the Croatian language to Discourse.

File: lib/translations_manager/locales.rb (hunk 1 above)
Comment (markodurdevic) [reply lib/translations_manager/locales.rb]: OK, changed that and amended the commit. Running `scripts/pull_translations.rb` in the main discourse repo threw an error when I removed `hr_HR`, but after deleting `config/locales/*hr_HR*` there, everything was fine.

**Comment by markodurdevic** [comment]
> Thanks for working on the translations! Are you going to create a PR to add the translations to Discourse core as well?

Sure, no problem. I just add the croatian locale files pulled by `pull_translations.rb` to the PR?

> Please ping me when the translation progress for `core/server.yml` reaches roughly 50% and I'll be happy to help you add the Croatian language to Discourse.

OK, tnx :)

**Comment by SamSaffron** [comment]
Just following up here @markodurdevic how is the translation of Discourse going?

**Comment by markodurdevic** [comment]
Unfortunately, I haven't had the time to work on it more, but I planned to take some time during the holidays to at least translate the server strings up to 60%. Maybe the pull request should be closed until this is ready?

**Comment by ZogStriP** [comment]
Thanks for the hard work @markodurdevic. Feel free to re-open a PR when the translations are ready 👍

**Pull request closed without merging**